documentation/plugins/index.markdown

```diff
@@ -11,3 +11,214 @@ title: Plugins
+{
+  display_path_ = node_handle_.advertise<moveit_msgs::DisplayTrajectory>(planning_pipeline::PlanningPipeline::DISPLAY_PATH_TOPIC, 10, true);
+  cartesian_path_service_ = root_node_handle_.advertiseService(CARTESIAN_PATH_SERVICE_NAME, &MoveGroupCartesianPathService::computeService, this);
+}
+</PRE>
+
+Moveit offers different [default capabilities](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_ros/move_group/src/default_capabilities). A description of these can be found [here](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/move_group/default_capabilities_plugin_description.xml).
+
+<a name="OccupancyMapUpdater"></a>
+## occupancy_map_monitor::OccupancyMapUpdater
+
+### Usage
+
+### Interface Description
+
+### Concrete Implementation
+
+<a name="PlanningRequestAdapter"></a>
+## planning_request_adapter::PlanningRequestAdapter
+
+### Usage
+
+### Interface Description
+
+### Concrete Implementation
+
+<a name="KinematicsBase"></a>
+## kinematics::KinematicsBase
+The KinematicsBase interface enables you, to:
+
+- write your own forward and inverse kinematic solvers
+- wrap external kinematic solver libraries
+
+### Interface Description
+The well documented interface is located [here](https://github.com/ros-planning/moveit/blob/3464e3d27bd8655aa6187ca5d3a031a4f72663e2/moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h).
+### Concrete Implementations
+Examples for implementations of the KinematicsBase are the KDL plugin ([header](https://github.com/ros-planning/moveit/blob/4ac0c7432d335f57aab6836cbcaaac3fccf4b6f9/moveit_ros/planning/kdl_kinematics_plugin/include/moveit/kdl_kinematics_plugin/kdl_kinematics_plugin.h), [cpp](https://github.com/ros-planning/moveit/blob/4ac0c7432d335f57aab6836cbcaaac3fccf4b6f9/moveit_ros/planning/kdl_kinematics_plugin/src/kdl_kinematics_plugin.cpp)) or the SrvKinematicsPlugin ([header](https://github.com/ros-planning/moveit/blob/4ac0c7432d335f57aab6836cbcaaac3fccf4b6f9/moveit_ros/planning/srv_kinematics_plugin/include/moveit/srv_kinematics_plugin/srv_kinematics_plugin.h), [cpp](https://github.com/ros-planning/moveit/blob/4ac0c7432d335f57aab6836cbcaaac3fccf4b6f9/moveit_ros/planning/srv_kinematics_plugin/src/srv_kinematics_plugin.cpp)).
+
+<a name="PlannerManager"></a>
+## planning_interface::PlannerManager
+
+### Usage
+
+Base class for writing for MoveIt planner by implementing it you can provide your own planners.
+
+### Interface Description
+
+The interface is defined in [planning_interface.h](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h).
+
+### Concrete Implementation
+
+An examplary implemenation of a planner is the [OMPL Planner](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_planners/ompl/ompl_interface/src/ompl_planner_manager.cpp). 
+
+<a name="MoveItControllerManager"></a>
```

Comment:
Did you try using pure Markdown for your section links? according to [stackoverflow](https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown):

> Github automatically parses anchor tags out of your headers. So you can do the following: